### ISSUE TITLE
HQM Rewards - CrC

### ISSUE THREAD

**Issue by devanchya** [Haggle1996/RevolutionPack#435]
Rewards from HQM block.  Not regular loot, made in late CrC.

![2016-05-26_21 23 46](https://cloud.githubusercontent.com/assets/6947885/15594913/459ae880-2388-11e6-9d0a-1b3c92781deb.png)
![2016-05-26_21 23 49](https://cloud.githubusercontent.com/assets/6947885/15594914/459c0a26-2388-11e6-80f0-e74cfde680c1.png)

**Comment by Haggle1996** [comment]
These are already removed in the update. Thanks for posting the shots.

**Comment by ReikaKalseki** [comment]
_Potion Crystals_?

**Comment by Haggle1996** [comment]
They weren't in the normal quests rewards or progression, so they're not displayed as a possible reward when you're working on the quests. HQM has a loot bag feature -- where you get a bonus bag in addition to the intended quest rewards. I had copied the HQM quest file when I moved to developing REV|2, and although I deleted all the quests, I had inadvertently left these two crystals in one of the random loot bags, along with another loot bag containing some of the runes that can be broken off a pylon. The bags are random, so there's no guarantee a player will get them.

I have cleaned these out of REV|2's HQM and will be shortly pushing the update. I hope you understand this was completely unintentional, and I've gone over the HQM book with a fine-toothed comb to make sure there are no more surprises. The items were not used to change or gate progression, they were simply 'bonus' loot left in by someone who helped me with the quest book.
